x/mint/types/keys.go

```diff
@@ -21,4 +21,12 @@ const (
 	// Query endpoints supported by the minting querier.
 	QueryParameters      = "parameters"
 	QueryEpochProvisions = "epoch_provisions"
+
+	// key for creating a new module account
+	SubmoduleAccountKey = "submodule"
```

Comment:
should this have a meaningful name? if it's just submodule, can we remove it from the account name?

(I could see this being used for additional namespacing in the mint module, e.g. maybe "community" makes sense here?)

Reply:
done

Reply:
(used community)